phpunit.xml.dist

```diff
@@ -33,5 +33,6 @@ It's auto-generated by sonata-project/dev-kit package.
     <php>
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="KERNEL_CLASS" value="Sonata\AdminBundle\Tests\App\AppKernel" />
```

Comment:
This file is autogenerated and cannot be touched.

Please add
```
protected static function getKernelClass(): string
{
    return AppKernel::class;
}
``` 
instead in your MenuTest file if needed

Reply:
Done.